CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# Release 2.47.0 30-01-2024
+
+- Fix the OpenSSL vulnerabilities by rebuilding the release
+- Update the Bootstrap proto link in Envoy Filters README.md (#872)
+- Allow internal callers to inject RouteGenerator (#870)
+- Switch to use RouteGenerators in production (#869)
+- Implement all remaining RouteGenerators (#868)
```

Comment:
- Fix the OpenSSL vulnerabilities by rebuilding the release

The rest are user-invisible changes.